wagtail/admin/templates/wagtailadmin/shared/button.stories.tsx

```diff
@@ -0,0 +1,218 @@
+      <span className="icon-wrapper">
+        <svg className="icon icon-plus icon" aria-hidden="true">
+          <use href="#icon-plus" />
+        </svg>
+      </span>
+      button element
+    </button>
+
+    <h4>(small)</h4>
+
+    <a href="#" className="button button-small bicolor button--icon">
+      <span className="icon-wrapper">
+        <svg className="icon icon-plus icon" aria-hidden="true">
+          <use href="#icon-plus" />
+        </svg>
+      </span>
+      button link
+    </a>
+    <button type="button" className="button button-small bicolor button--icon">
+      <span className="icon-wrapper">
+        <svg className="icon icon-plus icon" aria-hidden="true">
+          <use href="#icon-plus" />
+        </svg>
+      </span>
+      button element
+    </button>
+
+    <h3>Icon buttons without text</h3>
+
+    <a href="#" className="button text-replace button--icon">
+      <svg className="icon icon-cog icon" aria-hidden="true">
+        <use href="#icon-cog" />
+      </svg>
+      button link
+    </a>
+    <button type="button" className="button text-replace button--icon">
+      <svg className="icon icon-cog icon" aria-hidden="true">
+        <use href="#icon-cog" />
+      </svg>
+      button element
+    </button>
+
+    <h4>(small)</h4>
+
+    <a href="#" className="button button-small text-replace button--icon">
+      <svg className="icon icon-cog icon" aria-hidden="true">
+        <use href="#icon-cog" />
+      </svg>
+      button link
+    </a>
+    <button
+      type="button"
+      className="button button-small text-replace button--icon"
+    >
+      <svg className="icon icon-cog icon" aria-hidden="true">
+        <use href="#icon-cog" />
+      </svg>
+      button element
+    </button>
+
+    <h3>Colour signifiers</h3>
+
+    <h4>Positive</h4>
+
+    <a href="#" className="button yes">
+      yes
+    </a>
+    <a href="#" className="button button-small yes">
+      yes
+    </a>
+
+    <h4>Negative</h4>
+
+    <a href="#" className="button no">
+      No
+    </a>
+    <a href="#" className="button button-small no">
+      No
+    </a>
+
+    <h3>
+      Buttons with internal loading indicators (currently only{' '}
+      <code>button</code> supported)
+    </h3>
+    <button type="button" className="button button-longrunning">
+      <svg className="icon icon-spinner icon" aria-hidden="true">
+        <use href="#icon-spinner" />
+      </svg>
+      Click me
+    </button>
+
+    <h4>Secondary</h4>
+    <button
+      type="button"
+      className="button button-secondary button-longrunning"
+    >
+      <svg className="icon icon-spinner icon" aria-hidden="true">
+        <use href="#icon-spinner" />
+      </svg>
+      Click me
+    </button>
+
+    <h4>Small</h4>
+    <button type="button" className="button button-small button-longrunning">
+      <svg className="icon icon-spinner icon" aria-hidden="true">
+        <use href="#icon-spinner" />
+      </svg>
+      Click me
+    </button>
```

Comment:
Keep the button-longrunning but also add the active state (with the button disabled) so that we can see both variants at once.